tests/pipelines/test_pipelines_common.py

```diff
@@ -1689,28 +1672,6 @@ def _test_xformers_attention_forwardGenerator_pass(
         if test_mean_pixel_difference:
             assert_mean_pixel_difference(output_with_offload[0], output_without_offload[0])
 
-    def test_progress_bar(self):
```

Comment:
Would actually test this once. We can add it here 
https://github.com/huggingface/diffusers/blob/main/tests/pipelines/test_pipeline_utils.py